config/initializers/flipper.rb

```diff
@@ -31,6 +31,12 @@
 
   actor.respond_to?(:created_at?) && actor.created_at < "2025-08-11".to_time
 end
+Flipper.register(:enterprise_created_after_2025_08_11) do |actor|
+  # This group applies to enterprises only, so we return false if the actor is not an Enterprise
+  next false unless actor.actor.instance_of? Enterprise
+
+  actor.respond_to?(:created_at?) && actor.created_at >= "2025-08-11".to_time
```

Comment:
That's a long time ago now. Do we still want to activate this retrospectively? We don't know if new enterprises are using it...